stryker/guides/legacy/stryker-3/angular.md

```diff
@@ -0,0 +1,58 @@
+# Angular
```

Comment:
This is just a copy of the old version, correct? We might want to get rid of it, since it is no longer correct. Or we should specify the Stryker version for every command.

Reply:
I would like to keep it around for people who are stuck on Angular <=8. I will note Stryker 3.x in this legacy guide.